ansible/Makefile

```diff
@@ -1,3 +1,7 @@
+ANSIBLE_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+CAILLEACH_DIR:=$(abspath ${ANSIBLE_DIR}/../../cailleach)
+ENV_DIR:=${CAILLEACH_DIR}/environments/${ENV}
```

Comment:
```suggestion
ENV_DIR ?= $(abspath ${ANSIBLE_DIR}/../../cailleach//environments/${ENV})
```

Reply:
The ? thing didn't work the way I thought it does. I have written a few ifs, let me know if it seems ok.